crates/rpc/rpc-server-types/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "reth-rpc-server-types"
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+description = "RPC server types and constants"
+
+[lints]
+workspace = true
+
+[dependencies]
+# reth
+reth-rpc.workspace = true
```

Comment:
it'd be great if we can do without.

no that we have a separate crate for constants we can move all constants from reth-rpc crate in here and then import in reth-rpc instead, inverting the dependency.